<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>836</width>
    <height>621</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>831</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="window_tabs">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>811</width>
      <height>551</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: dark;</string>
    </property>
    <property name="currentIndex">
     <number>5</number>
    </property>
    <widget class="QWidget" name="main_tab">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(221, 221, 221);</string>
     </property>
     <attribute name="title">
      <string>Главная</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_12">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>90</y>
        <width>641</width>
        <height>321</height>
       </rect>
      </property>
      <property name="title">
       <string>Ресторан</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>511</width>
         <height>141</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">padding-left: 20;
background-color: dark;</string>
       </property>
       <property name="text">
        <string>Добро пожаловать в программу ресторана</string>
       </property>
       <property name="margin">
        <number>40</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>591</width>
         <height>141</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">padding-left: 20;
background-color: dark;</string>
       </property>
       <property name="text">
        <string>Здесь вы можете посмотреть список столов,
 забронировать стол или cоздать счет,
 а так же проверить список клиентов</string>
       </property>
       <property name="margin">
        <number>40</number>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Клиенты</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>811</width>
        <height>521</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Список клиентов</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="clients_table">
         <column>
          <property name="text">
           <string>ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>ФИО</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Номер телефона</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Работники</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>811</width>
        <height>521</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Список работников</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="workers_table">
         <column>
          <property name="text">
           <string>ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>ФИО</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Номер телефона</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Должность</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Зарплата</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tables_tab">
     <attribute name="title">
      <string>Столы</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>801</width>
        <height>610</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Информация о столах</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tables_table">
         <column>
          <property name="text">
           <string>Номер</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Зал</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Количество мест</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Занят</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Ближайшая бронь</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Счета</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>811</width>
        <height>521</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Список счетов</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="bills_table">
         <column>
          <property name="text">
           <string>ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>id стола</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>id сотрудника</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Итоговая стоимость</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Время создания</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Статус оплаты</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Бронь</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>801</width>
        <height>21</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Создать бронь</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>801</width>
        <height>501</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="1">
        <widget class="QGroupBox" name="groupBox">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="title">
          <string/>
         </property>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>50</y>
            <width>401</width>
            <height>351</height>
           </rect>
          </property>
          <property name="title">
           <string>Введите данные</string>
          </property>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>40</y>
             <width>371</width>
             <height>41</height>
            </rect>
           </property>
           <property name="title">
            <string>Номер клиента</string>
           </property>
           <widget class="QTextEdit" name="client_reservation_input">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>20</y>
              <width>371</width>
              <height>21</height>
             </rect>
            </property>
           </widget>
          </widget>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>110</y>
             <width>371</width>
             <height>41</height>
            </rect>
           </property>
           <property name="title">
            <string>Номер стола</string>
           </property>
           <widget class="QTextEdit" name="table_reservation_input">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>20</y>
              <width>371</width>
              <height>21</height>
             </rect>
            </property>
           </widget>
          </widget>
          <widget class="QGroupBox" name="groupBox_5">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>180</y>
             <width>371</width>
             <height>41</height>
            </rect>
           </property>
           <property name="title">
            <string>Время</string>
           </property>
           <widget class="QDateTimeEdit" name="time_reservation_input">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>20</y>
              <width>371</width>
              <height>21</height>
             </rect>
            </property>
           </widget>
          </widget>
          <widget class="QPushButton" name="reservation_btn">
           <property name="geometry">
            <rect>
             <x>102</x>
             <y>280</y>
             <width>201</width>
             <height>32</height>
            </rect>
           </property>
           <property name="text">
            <string>Забронировать</string>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Счет</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>801</width>
        <height>541</height>
       </rect>
      </property>
      <property name="title">
       <string>Счет</string>
      </property>
      <widget class="QGroupBox" name="groupBox_7">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>50</y>
         <width>321</width>
         <height>431</height>
        </rect>
       </property>
       <property name="title">
        <string>Создание счета</string>
       </property>
       <widget class="QGroupBox" name="groupBox_8">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>271</width>
          <height>41</height>
         </rect>
        </property>
        <property name="title">
         <string>Номер столика</string>
        </property>
        <widget class="QTextEdit" name="table_check_input">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>371</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_9">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>100</y>
          <width>271</width>
          <height>41</height>
         </rect>
        </property>
        <property name="title">
         <string>Номер клиента</string>
        </property>
        <widget class="QTextEdit" name="client_check_input">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>371</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_11">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>170</y>
          <width>271</width>
          <height>41</height>
         </rect>
        </property>
        <property name="title">
         <string>Номер обслуживающего сотрудника</string>
        </property>
        <widget class="QTextEdit" name="worker_check_input">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>371</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QPushButton" name="create_bill_btn">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>310</y>
          <width>271</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Создать</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_13">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>50</y>
         <width>321</width>
         <height>461</height>
        </rect>
       </property>
       <property name="title">
        <string>Оплата счета</string>
       </property>
       <widget class="QGroupBox" name="groupBox_14">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>271</width>
          <height>41</height>
         </rect>
        </property>
        <property name="title">
         <string>Номер счета</string>
        </property>
        <widget class="QTextEdit" name="bill_search_field">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>371</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_16">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>370</y>
          <width>271</width>
          <height>41</height>
         </rect>
        </property>
        <property name="title">
         <string>Сумма заказа</string>
        </property>
        <widget class="QTextEdit" name="sum_bill_input">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>371</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QPushButton" name="search_bill_btn">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>90</y>
          <width>271</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Найти</string>
        </property>
       </widget>
       <widget class="QPushButton" name="close_bill_btn">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>420</y>
          <width>271</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Оплатить</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox_17">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>160</y>
          <width>271</width>
          <height>41</height>
         </rect>
        </property>
        <property name="title">
         <string>Название блюда</string>
        </property>
        <widget class="QTextEdit" name="dish_name_input">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>371</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_18">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>210</y>
          <width>271</width>
          <height>41</height>
         </rect>
        </property>
        <property name="title">
         <string>Стоимость</string>
        </property>
        <widget class="QTextEdit" name="dish_price_input">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>371</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QPushButton" name="push_dish">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>260</y>
          <width>271</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Добавить в чек</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>836</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCourse">
    <property name="title">
     <string>Course</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuCourse"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
